nessus-agent/windows/init.sls

```diff
@@ -6,6 +6,46 @@
 # In addiont, the sls file requires that the salt winrepo has been
 # configured with a `nessus agent` pkg definition
 
+{%- if (nessus.log_rotation_strategy) %}
```

Comment:
The parens are unnecessary... goes for each `if` statement...

Reply:
Okay.  I'll remove.